app/facade/foods_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodsFacade
+  def self.create_food(query)
+    json = FoodsService.get_food_data(query)
+    json[:foods].map do |data|
+      Food.new(data)
+    end.take(10)
```

Comment:
You can also limit it to 10 foods by adding the pageSize parameter to the API call. That would remove the need to add this ruby logic here